### ISSUE TITLE
[ACTIVE_PREQUAL] PR1644 changed-head current-corpus overlap prequalification

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1844]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SCIENTIFIC_CONTROL: #548
SWARM_WORKER_ID: 1844
STATUS: PENDING_POSTCLAIM_DIRECT_ARBITRATION
SEQUENTIAL_PACKAGE: 2
PRIOR_PACKAGE: PR1836 exact-head prequal terminalized CHANGES_REQUIRED and released before this claim
SWARM_LANE_KEY: AUDIT-B|PR1644|CHANGED-HEAD-PREQUAL|EXACTHEAD-13A78050|CURRENT-CORPUS-OVERLAP-V1
SELECTED_WORK_KIND: NONOWNING_CHANGED_HEAD_PREQUAL / INDEPENDENT_VERIFY / DATA-INTEGRITY
TARGET_PR: #1644
TARGET_EXACT_HEAD_AT_CLAIM: 13a78050cd6b464b81a5d7586fb34ad2f937f19b
LIVE_MAIN_AT_CLAIM: 2da11c62f066671a1122888ec116242502f4fb9c
EXACT_HEAD_SHARED_CI: 34800069980=SUCCESS
INCUMBENT_PRODUCT_LINEAGE: #1715 / lawful successor semantics; PR moved substantively at 2026-09-14T02:41:24Z so no stale takeover is attempted
PREVIOUS_INDEPENDENT_AUDIT: #1720 PASS only on old exact head 6588ba6ff553e982b9fad1421503786944fe6c2f and narrow immutable-source-sealing scope; it does not cover this changed head/current-corpus overlap code
PRODUCT_MUTATION: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false
LOCAL_FREE_ONLY: true

WHY THIS UNIT NOW:
- current head is terminal-green and introduces/updates the current retained post-G05/G06 rows -> canonical overlap-matcher bridge, directly on D06 global-overlap/future-training-exclusion closure;
- prior source-sealing audit #1720 predates this changed head and explicitly did not terminalize global current-corpus overlap or broad D06 authority;
- Product has moved recently, so this worker must not take Product ownership; read-only prequalification can still catch exact trust/composition defects before producer release without collision;
- exact-head and semantic searches immediately before claim found no current exact-head verifier for 13a78050... .

CANDIDATES COMPARED:
1. PR1644 changed-head current-corpus overlap bridge — SELECTED: green, high-leverage, previous audit stale for changed code, no exact-head verifier found.
2. PR1836 — already prequalified by this worker and handed to active owner #1833; repaired head must be fresh different-worker audited, so no self-duplication.
3. PR1336 / PR590 / PR1256 — exact heads red and actively owned; collision.
4. physical ArXiv/LangUK replay and other current corpus acquisition lanes — active-owned (#1846 and neighboring workers); collision.

OWNED SURFACE AFTER ARBITRATION WIN:
- READ-ONLY exact-head inspection of PR1644 current-corpus overlap executor/tests and exact upstream/downstream authority contracts;
- no Product mutation, no branch/PR creation, no CI rerun on unchanged green head, no merge, no broad D06 PASS while producer has not released the changed head;
- publish concrete HIGH/MEDIUM findings with minimum same-lineage repair contract, or a narrow PREQUAL_NO_NEW_DEFECT result preserving all unproven scientific gates.

ACCEPTANCE:
1. Mandatory direct OPEN issue collection newest-first 100/page and separate OPEN PR collection newest-first 100/page reach exact physical [] sentinels; earlier valid same-semantic verifier wins.
2. Rebind PR1644 exact head/state and CI after arbitration and immediately before any authoritative finding.
3. Verify current post-G05/G06 row schema projection into matcher is closed-world, strict-type, content-preserving and consumes one authenticated physical-byte snapshot.
4. Verify training JSONL physical identity/hash/materialization identity are independently rooted, not caller-authored/self-resealed, and the exact bytes checked are the bytes parsed/matched.
5. Verify canonical exact/near-overlap matcher semantics, source reservation identities and future-training exclusion evidence are not widened from candidate-authored inputs.
6. Review duplicate JSON keys, symlink/path substitution, TOCTOU, loaded implementation origin/checkout binding, malformed/extra rows, normalization aliases, duplicate/accounting/order, durable evidence/raw-payload leakage and atomic publication.
7. Preserve distinct gates: current-corpus overlap evidence must not imply project-history tokenizer/training exposure zero, future-training exclusion consumption, purpose-specific evaluation authority, EVAL-303 activation, optimized exposure, training, learned weights or final-test access without physical evidence.

TRUTH_BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW: mandatory postclaim direct OPEN issues + separate OPEN PR arbitration to exact empty sentinels, exact-head race refresh, then read-only changed-head trust/composition prequalification if this package wins.

**Comment by Oleksii-debug** [comment]
POSTCLAIM ARBITRATION: **WIN** for this exact non-owning prequalification package.

Direct current OPEN issues were enumerated newest-first at 100/page through physical `page=14 -> []`; separate current OPEN PRs through physical `page=7 -> []`. Post-sweep exact-semantic refresh returned only #1844. Immediately before findings, PR #1836 remains OPEN/non-draft/mergeable at exact head `c3219a368aa9fbdf3b8975dd296d3a968d7636a7`, base `109a2933cd4a9c9c62906acb8668423d531f3490`; shared CI run `34800429127` remains terminal FAILURE. Product ownership remains #1833; this package remains read-only/non-owning and will not mutate Product bytes or rerun unchanged red CI.

**Comment by Oleksii-debug** [comment]
TERMINAL PREQUAL RESULT — **CHANGES_REQUIRED / LEASE RELEASED** for exact head `c3219a368aa9fbdf3b8975dd296d3a968d7636a7`.

Formal producer handoff is PR #1836 comment `5658664967`.

Findings:
- `PREQUAL1836-001 HIGH`: exact trusted checkout SHAs are not bound to the running carrier / loaded PR940 implementation origins+bytes, so alternate/stale executable/imported code can mint a receipt claiming trusted Git lineage. Same-lineage repair contract supplied; downstream PR1821's repaired loaded-source binding is the existing pattern to reuse.
- `PREQUAL1836-002 CI BLOCKER`: shared CI `34800429127`, job `103841838849`, fails Ruff `UP012` at `tests/test_prepare_data232_ephemeral_handoff_v1.py:161`; use `b"canonical payload"` instead of UTF-8 `.encode()`.

Independently verified nonblocking scope: canonical PR940 inventory/payload validation is fully consumed; exact physical payload bytes are hashed as parsed; duplicate/missing/extra/tampered payload rows fail closed; JSON duplicate keys fail closed; output publication is true no-replace; top-level receipt schema is closed; current PR1821 handoff semantics are mechanically compatible; zero-credit scientific truth remains unchanged.

No Product mutation, merge, CI rerun, corpus/tokenizer/exposure/training authority promotion, final-test outcome read, paid compute, pretrained weights, or external LLM/API data-intelligence use occurred. #1844 releases this exact-head prequal lease now. Any repaired head requires a fresh different-worker exact-head audit.
